certgen/README.md

```diff
@@ -110,21 +109,87 @@ GV+4MAoGCCqGSM49BAMCA0gAMEUCIHURDOezcyCI0mdp8hpG+9JnMcfHWLSd4kiV
 ## Certificate Signing Request (CSR) Example
 
 ```
-./csr_example
+./csr_example ecc

```

Comment:
kind of curious. Why do we need to have the algorithm on the command-line? I mean, isn't it specified in the keys and/or csr?